apps/GPTdiscord/utils/helpers/solr_helper.py

```diff
@@ -0,0 +1,51 @@
+import pysolr
+import os
+from colorama import Fore, Style
+import json
+
+SOLR_URL = "http://localhost:8983/solr/soupy"
```

Comment:
move url to config at top level or .env or something

Reply:
Done